dockerfiles/stages/1-build-deps

```diff
@@ -49,13 +49,18 @@ RUN apt-get update \
     curl \
     file \
     git \
+    '(^lld-10$|^lld-11$)' \
```

Comment:
Unfortunately the `lld` package is:
1. not available everywhere (in particular, it does not exist on stretch: https://packages.debian.org/search?keywords=lld&searchon=names&suite=all&section=all)
2. in bionic, it refers to a completely obsolete version (6.1) even though newer versions are available with `lld-X` packages

And, also unfortunately, neither `lld-10` or `lld-11` are available on all distributions, hence the regex, and the symbolic links below since these packages do not add a `lld` executable.

Reply:
Ok understood, your solution looks solid given all of those constraints ✅